apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -253,7 +253,7 @@ export class PublishAction extends BaseRushAction {
         for (const change of orderedChanges) {
           if (change.changeType && change.changeType > ChangeType.dependency) {
             const project: RushConfigurationProject | undefined = allPackages.get(change.packageName);
-            if (project) {
+            if (project && !this._packageExists(project)) {
```

Comment:
>project && !this._packageExists(project)) [](start = 16, length = 41)

If the package already exists, we should probably print a big warning.  That is super weird and the end user needs to know about it.